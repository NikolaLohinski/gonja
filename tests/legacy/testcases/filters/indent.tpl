# indent 4
{{ simple.long_text|indent }}
# indent 2
{{ simple.long_text|indent(2) }}
# indent 2, first=true
{{ simple.long_text|indent(2, true) }}
# indent 2, blank=true
{{ simple.long_text|indent(2, false, true) }}
# indent 2, first=true, blank=true
{{ simple.long_text|indent(2, true, true) }}
