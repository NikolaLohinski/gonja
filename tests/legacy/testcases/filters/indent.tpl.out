# indent 4
This is a simple text.

    This too, as a paragraph.
    Right?

    Yep!

# indent 2
This is a simple text.

  This too, as a paragraph.
  Right?

  Yep!

# indent 2, first=true
  This is a simple text.

  This too, as a paragraph.
  Right?

  Yep!

# indent 2, blank=true
This is a simple text.
  
  This too, as a paragraph.
  Right?
  
  Yep!

# indent 2, first=true, blank=true
  This is a simple text.
  
  This too, as a paragraph.
  Right?
  
  Yep!

